bindings/nodejs/lib/constants.ts

```diff
@@ -0,0 +1 @@
+export const SHIMMER_TESTNET_BECH32_HRP = 'rms';
```

Comment:
Maybe also add the others and also the CoinType
```JS
export enum CoinType {
    IOTA = 4218,
    Shimmer = 4219,
}
```

Reply:
Or maybe something like this should be even in @iota/types

Reply:
I'll add it in the meantime, and we can remove it if it gets added to @iota/types.